components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-resources/api-resources.hbs

```diff
@@ -0,0 +1,86 @@
+{{js "js/resources.js"}}
+
+<div class="page-header">
+    <h4 class="lead">Resources Configuration</h4>
+</div>
+<div class="row">
+    <div id="api_designer">
+        <div>
+            <div class="resource_create col-xs-12 col-sm-12 col-md-8 col-lg-8">
+                <div class="form-group">
+                    <label class="col-sm-3 control-label">URL Pattern</label>
+                    <div class="col-sm-9">
+                        <div class="input-group">
+                            <span class="input-group-addon" id="resource_url_pattern_refix">{{@params.context}}/{{@params.version}}/</span>
+                            <input id="resource_url_pattern" type="text" placeholder="E.g., path/to/resource" class="resource_url_pattern form-control"/>
+                        </div>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-sm-3 control-label" for=""></label>
+                    <div class="col-sm-9">
+                        <table style="width:100%">
```

Comment:
Moved the inline styles to a css file - 52f15bce21dc13955d1d57d660143c4f3f7a2236